frontend/test/playwright/utils/test-fixture.ts

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line no-restricted-syntax
+import { test as base } from "@playwright/test"
+
+/**
+ * This file overwrites the default playwright test fixture
+ * to apply some global settings and route handling.
+ */
+export const test = base.extend({
+  page: async ({ page }, use) => {
+    // Intercept /api/event requests
+    await page.route("/api/event", (route) => route.abort())
+    // Intercept  provider APIs
+    await page.route("**.jamendo.com**", (route) => route.abort())
+    await page.route("**.freesound.**", (route) => route.abort())
```

Comment:
Just curious, why only these two?

Reply:
These were the only two that existed, my guess would be that the tests only use sample audio data from these providers but perhaps someone else on @WordPress/openverse-frontend can confirm.

Reply:
That's probably an excellent idea, I'll create a separate issue.